src/gurt/telemetry.c

```diff
@@ -3200,6 +3226,8 @@ d_tm_init_histogram(struct d_tm_node_t *node, char *path, int num_buckets,
 		rc = -DER_NO_SHMEM;
 		goto failure;
 	}
+	histogram->dth_fast_bucketing =
+	    (multiplier == 2 && (initial_width & (initial_width - 1)) == 0);
```

Comment:
to me it reads "set dth_fast_bucketing iff mult is 2 and least-significant-bit of initial_width is set, is that correct?

Reply:
I'll add a comment in a subsequent patch. The idea is easier to see with an example. Say `initial_width` is 256. As a bit pattern, that value looks like this: `0b100000000`. Then, if we subtract 1 from it, we get this pattern: `0b11111111`. If we AND those together, we get 0. This only works for numbers that are clean powers of 2. 

Even easier to see if we "stack" the two values:
```
100000000
 11111111
```

Reply:
When you do add the comment, it would be great to detail not only the meaning of the bitwise math (although maybe that would be most clarified by using a named helper function or macro), but also why fast bucketing only works in the case where everything is a power of two. I think detailing the principle behind it will be most helpful in the future.